src/test/resources/config.properties

```diff
@@ -0,0 +1,2 @@
+browser = chrome
+environment = dev
```

Comment:
This properties file should be located in the main/recourses directory since it's relevant to our driver factory and configuration